kuma/users/templates/account/email.html

```diff
@@ -8,7 +8,7 @@
 
     {% if user.emailaddress_set.all().exists() %}
 
-      <p>{{ _('{email} is your <em>primary</em> email address')|fe(email=user.email) }}</p>
+      <p>{{ _('%(email)s is your <em>primary</em> email address'|safe, email=user.email) }}</p>
```

Comment:
Like so? ``{{ _('%(email)s is your <em>primary</em> email address', email=user.email)|safe }}``